src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -277,6 +279,54 @@ def test_aks_create_with_ingress_appgw_addon(self, resource_group, resource_grou
                 'addonProfiles.ingressApplicationGateway.config.subnetCIDR', "10.232.0.0/16")
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_with_namespace_enabled(self, resource_group, resource_group_location="westus2"):
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name
+        })
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-namespace-resources --generate-ssh-keys'
```

Comment:
Please pre-generate the ssh key-value and assign it to the cluster like the following, otherwise the creation of ssh keys during concurrent testing may cause competition.

```suggestion
            'name': aks_name,
            'ssh_key_value': self.generate_ssh_keys()
        })
        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-namespace-resources --ssh-key-value={ssh_key_value}'
```

Please also modify the other two test cases. Thanks!

Reply:
I made this change, tests are still failing. I see the "cannot overwrite cassette" exception in several places, does this mean I need to rerecord all the tests? Even those which I have not changed?

Reply:
@FumingZhang could you please take a look at the pipelines again? I made the change as requested in all the three test cases I added, it is still failing. The tests which are failing seem to be unrelated to the changes I made, it is throwing the cannot overwrite cassette exception again. Thanks.

Reply:
Yes, not related to the changes in your PR.   

The api version used by network command in the failed test cases is updated. Will fix in PR [#4860](https://github.com/Azure/azure-cli-extensions/pull/4860).